orbit/pkg/packaging/windows_templates.go

```diff
@@ -159,10 +159,36 @@ var windowsWixTemplate = template.Must(template.New("").Option("missingkey=error
                   Return="check"
                   Impersonate="no" />                    
 
+   <SetProperty Id="CA_WaitOrbit"
+                 Before ="CA_WaitOrbit"
+                 Sequence="execute"
+                 Value='&quot;[POWERSHELLEXE]&quot; -NoLogo -NonInteractive -NoProfile -ExecutionPolicy Bypass Wait-Process -Name orbit -Timeout 30 -ErrorAction SilentlyContinue' />
```

Comment:
1. Should there be a stop orbit before this? (Maybe there is and I cannot find it.)
2. What happens if the 30 second timeout is hit?

Reply:
1. Stop already happened at this point, it was triggered by the SCM (service control management), which signaled the process to shutdown. The SCM would force stop the service after a 30 secs (default 30000 ms value) timeout if the hosting service process didn't shutdown already (service host process can request the SCM more time to shutdown through passing the SERVICE_STOP_PENDING flag via SCM callback, but orbit.exe is not doing that, so it will be forced terminated by the SCM). Installer code should not avoid force terminating at this point to avoid breaking the SCM process lifecycle management.
2. SCM will force terminate the process.